File: Pertemuan 6/BelajarNestedIf.java
import java.util.Scanner; 
public class BelajarNestedIf {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.print("Masukkan umur Anda : ");
        int umur = s.nextInt();
        if(umur <= 5 && umur>= 0) {
            System.out.println("Anda masih balita");
            System.out.print("Masukkan umur orang tua Anda : ");
            int umurOrangTua = s.nextInt();
            if(umurOrangTua < 15);{
                System.out.println("Sebaiknya anda meminta bantuan pengasuh anak.");
            } 
         } else if (umur > 5 && umur <= 19){
            System.out.println("Anda sudah remaja");
         } else if (umur > 19) {
            System.out.println("Anda sudah dewasa");
         } else {
            System.out.println("Umur anda tidak valid");
         }
    }
}

File: Pertemuan 6/BelajarNestedIf.py
umur = input ("Masukkan umur Anda : ")
umur = int(umur)

if (umur >= 0 and umur <=5):
    print("Anda balita")
    umur_orangTua = input ("Masukkan umur orang tua Anda : ")
    if (int(umur_orangTua)< 20):
        print ("Orang tua Anda perlu pengasuh.")
    else :
        print ("Anda dapat mengasuh anak Anda sendiri.")
elif (umur > 5 and umur <= 19):
    print("Anda remaja")
else: 
    print("Umur anda tidak valid.")